lua/packer.lua

```diff
@@ -784,6 +790,22 @@ packer.loader_complete = function(lead, _, _)
   return completion_list
 end
 
+--- Completion for listing snapshots in `snapshot_path`
+--- Intended to provide completion for PackerRollback command
+--- TODO: using vim.fn.readdir to get entries the snapshot directory
+packer.snapshot_complete = function(lead, _, _)
+  local completion_list = {}
+  local res = io.popen('ls ' .. config.snapshot_path, 'r')
+  for entry in res:lines() do
+    if vim.startswith(entry, lead) then
+      table.insert(completion_list, entry)
```

Comment:
Super minor nitpick: generally, we'd prefer (for consistency and small potential performance gains) to use the `table_name[#table_name + 1] = item` style of appending over `table.insert`.